crates/context/config/src/client/protocol/icp.rs

```diff
@@ -239,17 +239,14 @@ impl Network {
 
         match response {
             Ok(CallResponse::Response((data, _))) => {
-                // Now try to decode the actual data
-                match candid::decode_one::<Result<(), String>>(&data) {
-                    Ok(decoded) => {
-                        match decoded {
-                            Ok(()) => Ok(vec![]),                     // Return empty vec for success
-                            Err(err_msg) => Ok(err_msg.into_bytes()), // Return error message as bytes
-                        }
-                    }
+                match candid::decode_one::<Result<Vec<u8>, String>>(&data) {
+                    Ok(decoded) => match decoded {
+                        Ok(return_data) => Ok(return_data),
```

Comment:
Tested context creation, conrext invite, context join

Reply:
Eh, yeah.. the return values for mutate on context config are null, we should care instead about proxy, which returns `ICProposalWithApprovals`. Try that.

Otherwise, I'd expect this to pretty much look the same as above with query. No need for `decode_one`.